draft-ietf-quic-transport.md

```diff
@@ -4179,36 +4179,38 @@ Note:
 data to a peer.
 
 ~~~
-       o
-       | Create Stream (Sending)
-       | Create Bidirectional Stream (Receiving)
-       v
-   +-------+
-   | Ready | Send RST_STREAM
-   |       |-----------------------.
-   +-------+                       |
-       |                           |
-       | Send STREAM /             |
-       |      STREAM_BLOCKED       |
-       v                           |
-   +-------+                       |
-   | Send  | Send RST_STREAM       |
-   |       |---------------------->|
-   +-------+                       |
-       |                           |
-       | Send STREAM + FIN         |
-       v                           v
-   +-------+                   +-------+
-   | Data  | Send RST_STREAM   | Reset |
-   | Sent  |------------------>| Sent  |
-   +-------+                   +-------+
-       |                           |
-       | Recv All ACKs             | Recv ACK
-       v                           v
-   +-------+                   +-------+
-   | Data  |                   | Reset |
-   | Recvd |                   | Recvd |
-   +-------+                   +-------+
+   o
+   | Create Bidirectional Stream (Peer-initiated)
```

Comment:
The decision we reached before was to make the same event trigger two transitions: one to create, one to move to "Send".

Reply:
Thank you for pointing that out. Reverted the changes to the diagram and updated the text in fc0f3e6.